cmd/create.go

```diff
@@ -127,13 +129,17 @@ func RunCreate(options *CreateOptions) error {
 
 	// Create our cluster resource
 	name := options.Name
-	var cluster *cluster.Cluster
+	var newCluster *cluster.Cluster
 	if _, ok := profileMapIndexed[options.Profile]; ok {
-		cluster = profileMapIndexed[options.Profile].profileFunc(name)
+		newCluster = profileMapIndexed[options.Profile].profileFunc(name)
 	} else {
 		return fmt.Errorf("Invalid profile [%s]", options.Profile)
 	}
 
+	if newCluster.Cloud == cluster.CloudGoogle && options.CloudId == "" {
+		return fmt.Errorf("--cloudid is required for google cloud.")
```

Comment:
So we don't want to assume that user is calling from the command line, as this will be ran as a controller in a pod in the future. Can we please update this to suggest the name of the field, and not a command line flag.

```go
    return fmt.Errorf("CloudID is required for Google Cloud")
```